install/kubernetes.md

```diff
@@ -9,40 +9,53 @@ Install Percona XtraDB Cluster on Kubernetes
    ```
    **Note:** *It is crucial to specify the right branch with `-b` option while cloning the code on this step. Please be careful.*
 
-1. The next thing to do is to add the `pxc` namespace to Kubernetes, not forgetting to set the correspondent context for further steps:
+1. Now Custom Resource Definition for PXC should be created from the `deploy/crd.yaml` file. Custom Resource Definition extends the standard set of resources which Kubernetes “knows” about with the new items (in our case ones which are the core of the operator). 
+
+   This step should be done only once; it does not need to be repeated with the next Operator deployments, etc.
+
+   ```bash
+   $ kubectl apply -f deploy/crd.yaml
+   ```
+
+   **Note:** *This step requires your user to have cluster-admin role privileges. For example, those using Google Kubernetes Engine can grant user needed privileges with the following command:* `$ kubectl create clusterrolebinding cluster-admin-binding1 --clusterrole=cluster-admin --user=<myname@example.org>`
```

Comment:
GKE has a different approach, it allows create CRD without admin priv, but rbac has mapping to Google Cloud roles, so it is needed cluster admin for RBAC apply.
in short move this line to rbac step